webpack.config.cjs

```diff
@@ -45,15 +45,7 @@ const commonConfig = {
             loader: 'postcss-loader',
             options: {
               postcssOptions: {
-                plugins: [
-                  [
-                    'tailwindcss',
-                    {
-                      config: './packages/extension/tailwind.config.cjs',
-                    },
-                  ],
-                  ['autoprefixer'],
-                ],
+                config: path.resolve(__dirname, './postcss.config.cjs'),
```

Comment:
Is it required by the storybook to move it from Webpack config to a separate `postcss.config.cjs` file?

Reply:
Got it